geomesa-spark/geomesa-spark-sql/src/main/scala/org/apache/spark/sql/SQLRules.scala

```diff
@@ -196,10 +234,11 @@ object SQLRules extends LazyLogging {
 
           // split up conjunctive predicates and extract the st_contains variable
           val (scalaUDFs: Seq[Expression], otherFilters: Seq[Expression]) = extractScalaUDFs(f)
+          val sparkFilters: Seq[Expression] = otherFilters ++ scalaUDFs
 
-          val (gtFilters: Seq[GTFilter], sFilters: Seq[Expression]) = scalaUDFs.foldLeft((Seq[GTFilter](), otherFilters)) {
+          val (gtFilters: Seq[GTFilter], sFilters: Seq[Expression]) = sparkFilters.foldLeft((Seq[GTFilter](), Seq[Expression]())) {
```

Comment:
If we aren't operating on just the scalaUDFs anymore, do we need to split the Filters into UDFs and not-UDFs?  

I.e., can we skip calling extractScalaUDFs?  Preferring instead to call only splitConjunctivePredicates(f)?